src/components/DocumentsPage/index.js

```diff
@@ -0,0 +1,122 @@
+import React, { Component } from 'react';
+// components
+import DocumentsEditor from './components/DocumentsEditor';
+import DocumentsManager from './components/DocumentsManager';
+// styles 
+import './styles.css';
+
+class DocumentsPage extends Component {
+  
+  constructor(props) {
+    super();
+    this.state = { documents: {}, document: null, unsaved: {} }
+  };
+
+  next_id = 0;
```

Comment:
better keep these inside of state as well

Reply:
But its update should not re-render component. Why it's better?